pynetdicom/apps/common.py

```diff
@@ -505,13 +505,19 @@ def setup_logging(args, app_name):
 
     # Setup pynetdicom library's logging
     pynd_logger = logging.getLogger("pynetdicom")
-    handler = logging.StreamHandler()
-    handler.setFormatter(formatter)
-    pynd_logger.addHandler(handler)
-    pynd_logger.setLevel(logging.ERROR)
+    _hasStreamHandler = False
+    for _handler in pynd_logger.handlers:
+        _hasStreamHandler = _hasStreamHandler or isinstance(_handler, logging.StreamHandler)
+    if not _hasStreamHandler: 
```

Comment:
Can just replace this with:
```python 
if not any(isinstance(h, logging.StreamHandler) for h in pynd_logger.handlers):
```

Reply:
Thank you for your review. I learned something new.

I fixed and pushed. Please check it.